src/components/rich-text/rich-text.tsx

```diff
@@ -0,0 +1,15 @@
+export interface RichTextProps
+    extends React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement> {
+    children: string | undefined;
+}
+
+export const RichText = ({ children, ...rest }: RichTextProps) => {
+    return children ? (
+        <div
+            {...rest}
+            dangerouslySetInnerHTML={{
```

Comment:
but isn't this dangerous ?
this is not a content which we generate, but rather our users. I understand that the most dangerous are users of users, but still.

However if Wix is doing same, than it is OK by me

Reply:
There a safer way to inject html from rich text content save by the user...this is pure xss depend on where the data is generated and fetched. maybe we want to use https://github.com/cure53/DOMPurify

Also where do win uses this...I want to see and hack

Reply:
wait...this description is site owner input maybe we don't care as much but then I would call this component something else including the word unsafe

Reply:
it will be better. but I would consider to have the safe comp any how. maybe not for this.
it's still an attack vector to inject a product to the db with xss